dot/services.go

```diff
@@ -419,6 +419,7 @@ func newSyncService(cfg *Config, st *state.Service, fg sync.FinalityGadget, veri
 		BabeVerifier:       verifier,
 		BlockImportHandler: cs,
 		MinPeers:           cfg.Network.MinPeers,
+		MaxPeers:           cfg.Network.MaxPeers,
```

Comment:
Is max peers to prevent someone spamming peer connection requests, or is there more reasons for this upper bound?

Reply:
yeah it's kinda dangerous if we have unlimited as we can get bogged some if someone spams us. we also don't need *that* many peers necessarily for a healthy network :p